mits/models.py

```diff
@@ -0,0 +1,179 @@
+from mits import *
+
+
+@Session.model_mixin
+class SessionMixin:
+    def logged_in_mits_team(self):
+        try:
+            return self.mits_team(cherrypy.session['mits_team_id'])
+        except:
+            raise HTTPRedirect('../mits_applications/team')
```

Comment:
Can we put a return_to type thing here? I hate going to a bookmark, being kicked back to a login screen, and then being dumped on the homepage.

Reply:
In this case we really can't do that because there is no login screen for them to log into.  Instead, the way that they continue their application is just by clicking the link we give them in their email, i.e. doing what we do on the ``confirm`` page where we look them up by id.

However, your suggestion did make me think that instead of simply redirecting back to the main page, we should instead redirect to a special page explaining that they should check their email and use the link we provided there, since that will probably save some confusion.  I've just added that page (see ``login_explanation.html`` below).